cmsimple/functions.php

```diff
@@ -2369,6 +2369,25 @@ function XH_registeredEditmenuPlugins()
     return XH_registerPluginType('editmenu');
 }
 
+/**
+ * Returns the names of the registered languagemenu plugins.
+ *
+ * @return string
+ *
+ * @since 1.7.6
+ */
+function XH_registeredLanguagemenuPlugins()
+{
+    $metaLangMenus = 'enum:standard,';
+    $regLangMenus = XH_registerPluginType('languagemenu');
+    foreach ($regLangMenus as $regLangMenu) {
+        $metaLangMenus .= $regLangMenu . ',';
+    }
+    $metaLangMenus = rtrim($metaLangMenus, ',');
```

Comment:
To avoid the last line, you could do:
```suggestion
    $metaLangMenus = 'enum:standard';
    $regLangMenus = XH_registerPluginType('languagemenu');
    foreach ($regLangMenus as $regLangMenu) {
        $metaLangMenus .= "," . $regLangMenu;
    }
```
However, I think I'd prefer using `implode()` (although I'm not keen using `array_unshift()`):
```suggestion
    $metaLangMenus = ;
    $regLangMenus = XH_registerPluginType('languagemenu');
    array_unshift($regLangMenus, 'enum:standard');
    $metaLangMenus = implode(',', $regLangMenus);
```